.devops/code-review-pipelines.yml

```
@@ -11,7 +11,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-4303-update-commands-docker-image
```

```suggestion
      ref: main
```

TODO to replace after dependent PR merge